cluster/manifests/psp/pod_security_policy.yaml

```diff
@@ -33,4 +33,10 @@ spec:
   supplementalGroups:
     rule: RunAsAny
   volumes:
-  - '*'
+  - emptyDir
+  - awsElasticBlockStore
```

Comment:
do we need `awsElasticBlockStore`? isn't this working via `persistentVolumeClaim`?